apps/api/src/donations/donations.controller.ts

```diff
@@ -42,7 +43,20 @@ export class DonationsController {
 
   @Post('create-checkout-session')
   @Public()
-  createCheckoutSession(@Body() sessionDto: CreateSessionDto) {
+  async createCheckoutSession(@Body() sessionDto: CreateSessionDto) {
+    if (sessionDto.mode == 'subscription' && (sessionDto.personId == null || sessionDto.personId.length == 0)) {
```

Comment:
It's a best practice to use strict comparison checks using `===` instead of `==`

Read more at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#strict_equality_using

```suggestion
    if (sessionDto.mode === 'subscription' && (sessionDto.personId === null || sessionDto.personId.length === 0)) {
```

Reply:
Thanks.
Only in Javascript you can have 2 == "2" as true.